packages/react-components/react-nav-preview/library/src/components/Nav/useNav.ts

```diff
@@ -151,7 +157,7 @@ export const useNav_unstable = (props: NavProps, ref: React.Ref<HTMLDivElement>)
       { elementType: 'div' },
     ),
     selectedValue,
-    selectedCategoryValue,
+    selectedCategoryValue: controlledSelectedCategoryValue,
```

Comment:
Won't this always pass through the prop and prevent any internal control updates? Should it be 'selectedValue'?